.circleci/config.yml

```diff
@@ -116,8 +116,8 @@ workflows:
         jobs:
             - check_code_quality
             - run_dataset_script_tests_pyarrow_latest
-            - run_dataset_script_tests_pyarrow_1
+            - run_dataset_script_tests_pyarrow_3
```

Comment:
You should undo this edit too, to fix CI.

Reply:
Whoops indeed, thanks